recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
+   md5: 951beb9a826c0c02fd6c08571b4365d2
+
+test:
+ command:
+  - run.sh test.vcf out logs k_out k_log 2 5 ab
```

Comment:
You might want to check for output here. I don't know exactly how snmf works, but it looks to me like the build script creates `run.sh` to have hard-coded arguments passed to `snmf.sh`.

That is, no matter what arguments you give to `run.sh`, it will always call `"snmf.sh vcf output logs best_k_output best_k_logfile kmin kmax groups threshold_group"`